package.json

```diff
@@ -58,7 +58,8 @@
     "classnames": "^2.2.3",
     "lodash": "^4.17.4",
     "moment": "^2.14.1",
-    "prop-types": "^15.5.7"
+    "prop-types": "^15.5.7",
+    "react-transition-group": "^1.1.3"
   },
```

Comment:
This isn't needed, UT I'll clean it up after the merge. Here is the reason why: react-transition-group isn't used directly by this project. Reactstrap uses it and has it listed as a peerDep. It was a recent change on their end which really only affects development of this library since reactstrap is a peerDep and needs to be installed as a devDep.